packages/material-ui/src/useAutocomplete/useAutocomplete.js

```diff
@@ -67,6 +67,7 @@ export default function useAutocomplete(props) {
     autoHighlight = false,
     autoSelect = false,
     blurOnSelect = false,
+    disabled,
```

Comment:
To fix the ESLint error.

```suggestion
    disabled: disabledProp,
```